src/Particle/HDFWalkerInput_0_4.h

```diff
@@ -57,9 +58,9 @@ struct HDFWalkerInput_0_4
   //propoery list to handle parallel io
   hid_t h_plist;
   //the last file which was processed
-  std::string FileName;
+  std::filesystem::path FileName_noext;
   //list of files to process
-  std::stack<std::string> FileStack;
+  std::stack<std::filesystem::path> FileStack;
```

Comment:
Does each element include extension or not? No need of updating the variable name, updating the comments can be sufficient.

Reply:
No, if the extension `.config.h5` is present it is removed. I updated the comments to reflect this.